### PR TITLE
Add OnFinality RPC Endpoint for Kava to chain.json

### PR DESCRIPTION
**Pull request by deverka**

### CODE REVIEW

**Review by ankurdotb — CHANGES_REQUESTED** [review]
The structure is fine, but if I hit https://kava.api.onfinality.io/public/status I get the following error message:

`{"jsonrpc":"2.0","id":"0","error":{"code":-32603,"message":"error processing the request: Error parsing relay: Error unmarshalling relay as json"}}`

Is this RPC endpoint functional?

**Comment by deverka** [comment]
Yes, it is functional. Let me cross check with the team

**Comment by hugh-onf** [comment]
@ankurdotb this endpoint only supports EVM RPC
`curl -X POST https://kava.api.onfinality.io/public -d '{"id": 1,"jsonrpc": "2.0","method": "eth_blockNumber"}' -H 'Content-Type: application/json'`

image
Would it be an issue?

Thanks.

File: kava/chain.json [hunk 1]
```diff
@@ -113,6 +113,10 @@
       {
         "address": "https://rpc-kava-01.stakeflow.io",
         "provider": "Stakeflow"
+      },
+      {
+        "address": "https://kava.api.onfinality.io/public",
```
Comment (dylanschultzie) [inline kava/chain.json:118]: I believe this should be moved to line 171 under `evm`

**Review by dylanschultzie — CHANGES_REQUESTED** [review]

**Comment by deverka** [comment]
PR udpated! Sorry for a late response.

**Pull request merged**